lib/membrane_element_rtp_aac/payloader.ex

```diff
@@ -0,0 +1,57 @@
+defmodule Membrane.RTP.AAC.Payloader do
+  @moduledoc "TODO"
+
+  use Membrane.Filter
+
+  alias Membrane.Buffer
+  alias Membrane.{AAC, RTP}
+  alias Membrane.RTP.AAC.Utils
+
+  def_input_pad :input, accepted_format: %AAC{encapsulation: :none}
+  def_output_pad :output, accepted_format: %RTP{}
+
+  def_options bitrate: [
+                spec: :lbr | :hbr
+              ],
+              frames_per_packet: [
+                spec: pos_integer()
+              ]
+
+  @impl true
+  def handle_init(_ctx, options) do
+    state =
+      options
+      |> Map.from_struct()
+      |> Map.put(:acc, [])
+
+    {_actions = [], state}
```

Comment:
[NIT] I know it used to be written that way in this repo, but I don't think we need to emphasise that the first argument is the list of actions, so you can change it:
```suggestion
    {[], state}
```